### DISCUSSION TITLE
Navigation between react pages and keycloak pages

### DISCUSSION THREAD

**Discussion by sarrahadjayache** [keycloakify/keycloakify#201] [Q&A]
Hello, I started recently reading the documentation of keycloakify and looked at both [demo](https://github.com/garronej/keycloakify-demo-app) and [starter](https://github.com/garronej/keycloakify-starter) projects.

I'm still beginner here and i'm a little confused 🥲 : 
My main goal is to customize a theme in keycloakify and add it in keycloak (not in my local ) and make simple navigation between home page from react to login pages from keycloak.

- The real navigation between home screen and login keycloak is not really clear in the documentation unfortunately. I tried to look at this [repo](https://github.com/InseeFrLab/onyxia-web/blob/main/src/ui/components/App/App.tsx) but i couldn't set it up on my local machine to test it and i cannot find projects with similar case.

Thanks for providing us this solution anyway 😊

And thanks in advance

**Comment by garronej** [comment]
Hi @sarrahadjayache,  

> The real navigation between home screen and login keycloak is not really clear in the documentation unfortunately. I tried to look 
> at this [repo](https://github.com/InseeFrLab/onyxia-web/blob/main/src/ui/components/App/App.tsx) but i couldn't set it up on my 
> local machine to test it and i cannot find projects with similar case.  

Keep keep in mind that this repo is only for creating **theme** for the login and register pages of your Keycloak server.  
In the documentation I assume that users have Keycloak working already.   
What you describe, if I understand you well, isn't documented in the Keycloakify docs because it's out of the scope of Keycloakify.   

If I where you, I'd attempt to get login to your React native aplication via Keycloak working, forget about Keycloakify for now.  
Once you have a setup where user can create account and login then you can come back to Keycloakify and use it to customize the look and feel of your login/register page.  

Good luck
